### PR TITLE
[Backport][ipa-4-7] pkinit enable: use local dogtag only if host has CA

### PR DESCRIPTION
**Pull request by flo-renaud**
This PR was opened automatically because PR #2706 was pushed to master and backport to ipa-4-7 is required.

### CODE REVIEW

**Comment by flo-renaud** [comment]
PR was ACKed automatically because this is backport of PR #2706. Wait for CI to finish before pushing. In case of questions or problems contact @flo-renaud who is author of the original PR.

**Comment by tiran** [comment]
Please rebase to fix Travis CI. I just merged #2734

File: ipaserver/install/krbinstance.py [hunk 1]
```diff
@@ -45,6 +45,7 @@
 from ipaserver.install import ldapupdate
 
 from ipaserver.install import certs
+from ipaserver.masters import find_providing_servers
```
Comment (tiran) [inline ipaserver/install/krbinstance.py]: The 4.7 branch doesn't have the ``ipaserver.masters` module. The function is located in ``ipaserver.install.service``.

Comment (flo-renaud) [reply ipaserver/install/krbinstance.py]: Thanks for the hint, PR updated.

**Comment by tiran** [comment]
ipa-4-7:

* 20b0a7a96e13bbfa46efcd5b770a92390b3a5b50 pkinit enable: use local dogtag only if host has CA
* 36364600ef5ad75cc8d15315b521d2367ebc1adf ipatests: add integration test for pkinit enable on replica

**Pull request closed without merging**